xonsh/xontribs.py

```diff
@@ -44,13 +40,34 @@ def xontrib_context(name):
     return ctx
 
 
+def prompt_xontrib_install(names):
+    """Returns a formatted string with name of xontrib package to prompt user"""
+    md = xontrib_metadata()
+    packages = []
+    for name in names:
+        for xontrib in md['xontribs']:
+            if xontrib['name'] == name:
+                packages.append(xontrib['package'])
+
+    print('The following xontribs are enabled but not installed: \n'
+          '   {xontribs}\n'
+          'To install them run \n'
+          '    pip install {packages}'.format(xontribs=' '.join(names),
+                                               packages=' '.join(packages)))
```

Comment:
The xontribs actually list how they are installable in the metadata for the package that they are part of.  So this message need not strictly be about pip.  However, I think that can be fixed up in another PR.